### PR TITLE
Data column explanation + updating tenants

### PR DESCRIPTION
**Pull request by tomcrofty**
I've clarified how the data column works along with how to update a tenant's custom column and an attribute in its data column. I've also cleared up how the getCustomColumns() function works (When I first used the package I thought it meant to define columns that will be used in the data column).

### CODE REVIEW

File: source/docs/v3/tenants.blade.md [hunk 1]
```diff
@@ -60,17 +59,29 @@ After the tenant is created, an event will be fired. This will result in things
...
+    'customAttribute' => 'value', // to be store in the `data` JSON column
+    'plan' => 'free' // to be stored in the plan column (see below)
+]);
+```
+or simply
+```php
+$tenant->customAttribute = 'value'; // to be store in the `data` JSON column
+$tenant->plan = 'free'; // to be stored in the plan column (see below)
+$tenant->save();
+```
```
Comment (stancl) [inline source/docs/v3/tenants.blade.md:75]: Your use of *custom attribute* is the opposite here. Custom columns have their own columns.

File: source/docs/v3/tenants.blade.md [hunk 2]
```diff
@@ -60,17 +59,29 @@ After the tenant is created, an event will be fired. This will result in things
...
 
-You may define the custom columns be overriding the `getCustomColumns()` method on your `Tenant` model:
+You may define the custom columns that **won't** be used in the `data` JSON column by overriding the `getCustomColumns()` method on your `Tenant` model:
```
Comment (stancl) [inline source/docs/v3/tenants.blade.md:77]:
```suggestion
You may define the custom columns that **won't** be stored in the `data` JSON column by overriding the `getCustomColumns()` method on your `Tenant` model:
```

File: source/docs/v3/tenants.blade.md [hunk 3]
```diff
@@ -97,7 +108,7 @@ Also a good rule of thumb is that when you need to query the data with `WHERE` c
 
 ## Running commands in the tenant context {#running-commands-in-the-tenant-context}
 
-You may run commands in a tenant's context and then return to the previous context (be it central, or another tenant's) by passing a callable to the `run()` method on the tenant object. For example:
+You may run commands in a tenant's context (e.g. creating a user in the tenants user database) and then return to the previous context (be it central, or another tenant's) by passing a callable to the `run()` method on the tenant object. For example:
```
Comment (stancl) [inline source/docs/v3/tenants.blade.md:111]: I don't think this is needed here considering the code example.

File: source/docs/v3/tenants.blade.md (hunk 1 above)
Comment (lukinovec) [inline source/docs/v3/tenants.blade.md:75]: Suggestion:

Attributes of the tenant model which don't have their column will be stored in the `data` JSON column. You can set these attributes as you'd normally set model attributes:

```php
$tenant->update([
    'attributeThatHasNoColumn' => 'value', // stored in the `data` JSON column
    'plan' => 'free' // stored in the `plan` column (see below)
]);
```
or
```php
$tenant->customAttribute = 'value'; // stored in the `data` JSON column
$tenant->plan = 'free'; // stored in the `plan` column (see below)
$tenant->save();
```

**Pull request closed without merging**